### PR TITLE
Milestone: 3

### PR DESCRIPTION
**Pull request by ey-charles**
- Used Bootstrap for layout styling and basic functionality
- Applied custom classes for colors and backgrounds
- Finished topbar menu and footer section
- Used semantic html elements
- Included the three main grid columns using flexbox
- Used Bootstrap classes for styling

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+.t-font-roboto {
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+.t-btn {
+  color: black;
+  font-weight: 700;
+  font-size: 13px;
+  background: #fee400;
+  border: none;
```
Comment (BrittanyBlake) [inline main.css]: - [x] This can be done with the class `border-0`

File: main.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  border-radius: 15px;
+  padding-left: 0.75rem;
+  padding-right: 0.5rem;
+  text-transform: uppercase;
```
Comment (BrittanyBlake) [inline main.css]: - [x] This can be done with the bootstrap class `text-uppercase`

File: main.css [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+.topbar-logo svg {
+  max-width: 100%;
+}
+
+.topbar-icon {
+  font-size: 1.5rem;
+}
+
+.topbar-login {
+  text-transform: uppercase;
```
Comment (BrittanyBlake) [inline main.css]: - [x] This can be done with the bootstrap class `text-uppercase`

File: main.css [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+ * Page navigation
+ * ======================
+ **/
+.pagenav-item {
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  border-right: 1px solid #d9d9d9;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (BrittanyBlake) [inline main.css]: - [x] This can be done with the bootstrap classes `d-flex`, `align-items-center` and `justify-content-center`

File: main.css [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+  color: #f72211;
+}
+
+.pagenav-item > li {
+  border: none;
+}
+
+.pagenav-search-container {
+  border: none;
+}
```
Comment (BrittanyBlake) [inline main.css]: - [x] This can be done with the bootstrap class `border-0`

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @ey-charles and @JuanDavid31! 👋 

You have done an amazing job with this milestone! It looks really good 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Some of your css styles can be done with bootstrap classes. Take a look at your css file again and use this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) to see what can be done with bootstrap instead. 

****
### Please note that milestone 1 does not require a re-review and will be marked as approved in your dashboard, however these changes are MANDATORY. Please make the required changes before moving on to Milestone 2🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: main.css (hunk 1 above)
Comment (ey-charles) [reply main.css]: Thank you for your feedback!

**Review by ghost — COMMENTED** [review]
# Status: Invalid

**Hello team, I marked this milestone as invalid because it doesn't have a descriptive README file. You can use the Microverse readme [template](https://github.com/microverseinc/readme-template) as a guideline.**

**Submit for another review after you have implemented the above change.**

## Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

File: main.css [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+.t-bg-verydark-gray {
+  background-color: #c2d4d7;
+}
+
+.t-bg-translucid {
+  background: rgba(255,255,255,.9);
+}
+
+.t-bg-white {
+  background-color: white;
```
Comment (uryelah) [inline main.css]: - [ ] Double check if all of your styles in this file really can't be done using Bootstrap classes instead.
Some styles you are currently using pure CSS to apply but could be easily exchanged for Bootstrap classes are:
- bg or text color white
- font-weight
- width 100%
- display
- position

Make a quick sweep on your stylesheets, check the classes you can use on the [Bootstrap documentation](https://getbootstrap.com/docs/4.1/getting-started/introduction/), and whenever possible use the Bootstrap class instead.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi **Carlos Castillo Velasquez** & **Juan Piza Walteros** , I'm Sarah your reviewer for this PR 👩‍🚀 

You guys did an outstanding job in this milestone!

There are still some points to work on before finishing this project so just check my feedback carefully, **make the required changes, and then start working on Milestone 3**!

Remember sure to use the same PR, and that re-reviews are not needed for Milestone 1 and 2 😉 

You did great so keep on coding and Rock on! 🎸 👨‍🎤 

# Required Changes 🏗️ 🐛 💡

-----

 **Kindly fix the pointed out items and comments below, then start working on the next milestone**
 
 - [x] Make sure your linters are passing, fix them before starting working on the next milestone.

- [x] Each article image should also be a link, the same way the titles are. The user of the NewsWeek site is able to click an image and be linked to the article, so make sure there are links around each article image.

- [ ] [OPTIONAL] If you want to be even more semantic, use the `article` tag for each article inside your clone sections. Read more on the [article tag here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).

----

- [x] Make sure the navbar links contain the grey dividers between each link and the grey border between the first white row and the second.
Refer to how the navbar should look on wide screens below:
header_mobile

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: main.css (hunk 6 above)
Comment (uryelah) [reply main.css]: By getting the full scope of what Bootstrap can do now you'll be able to style your future projects with it way faster, so doing a little digging in the documentation right now will pay really pay off.

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [x] Do not commit files that are not related to the project. Instead, add them to `.gitignore` file. For instance, the `.DS_Store` file needs to be moved to the gitignore.
- [x] The main Navigation menu below the Navbar should have a form element with a search input. Note that it should be a form element.
- [x] The Icons in the footer should have links
- [x] All the footer nav-links should have links.

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**